runtime/docker/docker.go

```diff
@@ -380,6 +356,19 @@ func (c *DockerRuntime) ListContainers(ctx context.Context, labels []string) ([]
 	return c.produceGenericContainerList(ctrs, nr)
 }
 
+func (c *DockerRuntime) containerFilterBuilder(gfilters []*types.GenericFilter) filters.Args {
```

Comment:
let use the same name for these functions across docker.go and containerd.go?

like `buildFilterString`?

Reply:
have adjusted that